css/styles.css

```diff
@@ -7,6 +7,7 @@
 body {
   font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
     Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+  position: relative;
```

Comment:
you don't have to give all the elemente (header, main) "relative" if you aren't using "absolute" on a chil-element